python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_wingweight.py

```diff
@@ -0,0 +1,462 @@
+algo = ot.FunctionalChaosAlgorithm(
+    inputDesignPCE, outputDesignPCE, m.distributionX, adaptiveStrategy, projectionStrategy)
+
+algo.run()
+result = algo.getResult()
+print(result.getResiduals())
+print(result.getRelativeErrors())
+
+# %%
+# The relative errors are very low that indicate the good accuracy of the PCE model.
+
+# %%
+# Then, we exploit the surrogate model to compute the Sobol' indices.
+sensitivityAnalysis = ot.FunctionalChaosSobolIndices(result)
+print(sensitivityAnalysis.summary())
+firstOrder = [sensitivityAnalysis.getSobolIndex(i) for i in range(m.dim)]
+totalOrder = [sensitivityAnalysis.getSobolTotalIndex(
+    i) for i in range(m.dim)]
+graph = ot.SobolIndicesAlgorithm.DrawSobolIndices(
+    inputNames, firstOrder, totalOrder)
+graph.setTitle('Sobol indices by Polynomial Chaos Expansion - wing weight')
+view = otv.View(graph)
+
+# %%
+#
+# The Sobol' indices confirm the previous analyses, in terms of ranking of the most influent variables. We also see that five variables have a quasi null total Sobol' indices, that indicates almost no influence on the wing weight.
+# There is no discrepancy between first order and total Sobol' indices, that indicates no or very low interaction between the variables in the variance of the output.
+
+
+# %%
+# HSIC indices
+# ------------
+
+# %%
+# We then estimate the HSIC indices using a data-driven approach by exploiting the design of experiments used to train the PCE model.
+inputDesignHSIC = inputDesignPCE
+outputDesignHSIC = outputDesignPCE
+
+covarianceModelCollection = []
+
+# %%
+for i in range(m.dim):
+    Xi = inputDesignHSIC.getMarginal(i)
+    inputCovariance = ot.SquaredExponential(1)
+    inputCovariance.setScale(Xi.computeStandardDeviation())
+    covarianceModelCollection.append(inputCovariance)
+
+# %%
+# We define a covariance kernel associated to the output variable.
+outputCovariance = ot.SquaredExponential(1)
+outputCovariance.setScale(outputDesignHSIC.computeStandardDeviation())
+covarianceModelCollection.append(outputCovariance)
+
+#%%
+#
+# In this paragraph, we perform the analysis on the raw data: that is
+# the global HSIC estimator.
+
+estimatorType = ot.HSICUStat()
+
+# %%
+# We now build the HSIC estimator:
+globHSIC = ot.HSICEstimatorGlobalSensitivity(
+    covarianceModelCollection, inputDesignHSIC, outputDesignHSIC, estimatorType)
+
+# %%
+# We get the R2-HSIC indices:
+R2HSICIndices = globHSIC.getR2HSICIndices()
+print("\n Global HSIC analysis")
+print("R2-HSIC Indices: ", R2HSICIndices)
+
+# %%
+# and the HSIC indices:
+HSICIndices = globHSIC.getHSICIndices()
+print("HSIC Indices: ", HSICIndices)
+
+# %%
+# The p-value by permutation.
+pvperm = globHSIC.getPValuesPermutation()
+print("p-value (permutation): ", pvperm)
+
+# %%
+# We have an asymptotic estimate of the value for this estimator.
+pvas = globHSIC.getPValuesAsymptotic()
+print("p-value (asymptotic): ", pvas)
+
+# %%
+# We vizualise the results.
+graph1 = globHSIC.drawHSICIndices()

```

Comment:
I think if you give names to the columns of your input sample (using [setDescription](https://openturns.github.io/openturns/1.19/user_manual/_generated/openturns.Sample.html#openturns.Sample.setDescription)) these names will appear in the HSIC graphs.